### DISCUSSION TITLE
Attachements always go to amazon url, even though AWS_S3_UPLOAD_BUCKET_URL is pointing to vultr

### DISCUSSION THREAD

**Discussion by vsviridov** [outline/outline#7969] [Self Hosting]
It used to work before, but now it's trying to just go to an AWS url...

**Comment by tommoor** [comment]
It used to work before what?

**Reply by vsviridov** [reply]
I've created the attachment about a year ago, and have upgraded the outline version as they've become available. I'm not sure which version caused it.

**Reply by vsviridov** [reply]
<img width="1137" alt="image" src="https://github.com/user-attachments/assets/6e740dcf-a5ea-4f48-b6d0-c9bdc6e1b05c">

**Reply by tommoor** [reply]
Try setting `AWS_S3_FORCE_PATH_STYLE=true` perhaps

**Comment by vsviridov** [comment]
Tried that, that's how it was set originally
